src/grid/componentGrid.js

```diff
@@ -10,19 +10,23 @@ const ComponentGridComponent = props => {
         headerClass,
         rowClickHandler,
         style,
+        useDefaultStyle,
         rowsContainerStyle,
         rowHeader,
         rowFooter
     } = props;
 
+    const setDefaultStyle = (typeof useDefaultStyle === 'undefined') ? true : useDefaultStyle;
```

Comment:
This actually seems like it might be a good use of React's defaultProps concept: https://facebook.github.io/react/docs/react-component.html#defaultprops